lib/stripe/resources/dispute.rb

```diff
@@ -13,11 +13,5 @@ def close(params = {}, opts = {})
       resp, opts = request(:post, resource_url + "/close", params, opts)
       initialize_from(resp.data, opts)
     end
-
-    def close_url
-      resource_url + "/close"
-    end
-    extend Gem::Deprecate
-    deprecate :close_url, :none, 2019, 11
```

Comment:
Hmmmm we did bother to give a date and are still removing before that date, is that okay?

Reply:
I really don't think many people actively rely on the `*_url` methods so I think it's alright to remove them a bit sooner than we announced.

Reply:
==